### PR TITLE
Translated Hello World page

### PR DESCRIPTION
**Pull request by nsadev**
Hope everything works as it should, let me know if any changes are needed.

@tesseralis @rloqvist

### CODE REVIEW

File: content/docs/hello-world.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Ett litet React exempel ser ut så här:
```
Comment (rloqvist) [inline content/docs/hello-world.md]: The first part of the compound noun "React exempel" is a proper noun and thus it should probably be together with a hyphen as such "React-exempel". Refer to this page for some more details https://resurs.slu.se/samskriva-eller-sarskriva-one-word-or-two/

File: content/docs/hello-world.md [hunk 2]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Klicka på länken ovan för att öppna en kodredigerare online. Testa att göra ändringar i koden och se hur de påverkar innehållet som visas. De flesta sidor i den här guiden kommer att ha exempel som detta där du kan göra ändringar och testa dig fram.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Hur den här guiden fungerar {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+I den här guiden kommer vi att titta närmre på de byggblock som en React app består utav: element och komponenter. När du behärskar dem kan du skapa komplexa appar utifrån mindre byggnadsdelar som kan återanvändas.
```
Comment (rloqvist) [inline content/docs/hello-world.md]: Same thing here with "React app" -> "React-app"

File: content/docs/hello-world.md [hunk 3]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
+>Den här guiden använder till och från nyare JavaScript syntax i exemplen. Om du inte har jobbat med JavaScript under de senaste åren bör [dessa tre punkter](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) hjälpa dig på vägen.
 
 
-## Let's Get Started! {#lets-get-started}
-
-Keep scrolling down, and you'll find the link to the [next chapter of this guide](/docs/introducing-jsx.html) right before the website footer.
-
+## Nu kör vi igång! {#lets-get-started}
 
+Fortsätt scrolla, länken till [nästa kapitel i den här guiden](/docs/introducing-jsx.html)hittar du precis innan sidfoten.
```
Comment (rloqvist) [inline content/docs/hello-world.md]: I think there is a blank space missing just after the link.

**Comment by rloqvist** [comment]
Great work! I haven't had the chance to run this yet and look at the live version, but from what I've seen it should be fine. Take a look at the comments and the change request and we can go ahead and merge this :slightly_smiling_face:

File: content/docs/hello-world.md [hunk 4]
```diff
@@ -15,36 +15,34 @@ ReactDOM.render(
...
+React är ett JavaScript-bibliotek och därför kommer vi anta att du har en basal förståelse för JavaScript som programmeringsspråk. **Om du inte känner dig väldigt säker på din kunskapsnivå rekommenderar vi [att gå igenom en guide om JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) för att se hur du ligger till**, så att du kan följa med i guiden utan att tappa bort dig. Det kan ta mellan 30 min och en timme, men resultatet blir att du inte behöver känna det som att du lär dig både React och Javascript på samma gång.
 
->Note
+>Notera
 >
->This guide occasionally uses some of the newer JavaScript syntax in the examples. If you haven't worked with JavaScript in the last few years, [these three points](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) should get you most of the way.
+>Den här guiden använder till och från nyare JavaScript syntax i exemplen. Om du inte har jobbat med JavaScript under de senaste åren bör [dessa tre punkter](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) hjälpa dig på vägen.
```
Comment (rloqvist) [inline content/docs/hello-world.md]: "JavaScript syntax" -> "JavaScript-syntax"

**Comment by rloqvist** [comment]
@nsadev This is how the live version looks if you haven't gotten your environment up yet :slightly_smiling_face: 

hello-world-translate

**Comment by nsadev** [comment]
Thanks for spotting the things needing correction @rloqvist, was a bit unsure if without hyphen was right. Have corrected it all and added it.

Also thank you for the screenshot, great to see! Hope to get some time this weekend to take a better look at the problem with Gatsby to get the site visible.

**Comment by rloqvist** [comment]
Nice! Could you remove the file `hej-varlden.md` aswell :slightly-smiling-face:

**Comment by nsadev** [comment]
Sorry, didn't see it in the file tree. How do I create a commit when there's only a deleted file that's changed, add it as usual or something else?

**Comment by rloqvist** [comment]
That depends on what `git status` yields. If it shows that you deleted the file you can make a commit and push it.

**Comment by nsadev** [comment]
@rloqvist all ok with the latest?

**Pull request merged**